
### Members:
Alivia Blount
Kennan Salinero
Masha Aseeva
Nikhita Damaraju
Nitin Pasumarthy

### What we did

#### Generate pathway images with variant genes highlighted

We used the VCF file generated by Qiagen and applied additional filters to exclude common variants from 4 other databases.

1. A .vcf file filtering the Whole Genome Sequence by:
 1. Confidence: Call quality at least 20 (assay)
 2. Exclude "common" variants that are:
     1.  at least 1% in 1000 Genomes Project
     2.  at least 1% of all in the ExAC
     3.  at least 1% of all in the gnomAD
     4.  at least 1% of all NHLBI ESP exomes
     5. UNLESS it is confirmed Pathogenic
     
2. Next we used the Databse for Annotation, Visualisation and Integrated Discovery [(DAVID)](https://david-d.ncifcrf.gov/) to map gene names to generate a mapping file that maps gene name input from the above VCF to different ontologies / databases (gene names, GO, INTERPRO, KEGG). This produced a list of metabolic pathway names.

3. We generated a list of mutated genes for each KEGG pathway, and then fed it to R. The pathway images were generated using the ‘pathview’ package of Bioconductor in R. The package pulls in the KEGG pathway and labels the genes that I gave it.



